### ISSUE TITLE
Unpackaged file drop support

### ISSUE THREAD

**Issue by avaer** [webaverse/xrpackage#77]
`XRPackage` supports compiling XRPKs live from file contents, but the frontend currently only accepts pre-packaged things.

In the frontend, it would be useful to be able to drop something that isn't a package but can be converted into one (such as a VRM or GLTF), and have it be auto-packaged live so it can interact with the rest of the world.

**Comment by shu8** [comment]
xrpackage.org/edit.html seems to already support uploading non-wbn's and auto-packaging them in the browser using the `Add package` button (https://github.com/webaverse/xrpackage-site/blob/master/run.js#L196).

Is that what this issue is about, or was this issue about unpackaged file drop support somewhere else?

In any case I think it's worth clarifying in the UI of /edit.html that "Add package" means upload a wbn/gltf/glb/vrm/html file to add into the world -- we could rename it to "upload package" maybe? and it could open a panel that explains what you can do and then also add support for dragging a file into that panel instead of having to use the file picker

**Comment by avaer** [comment]
> Is that what this issue is about, or was this issue about unpackaged file drop support somewhere else?

Yeah that's what this issue is about. I guess that is supported now, but not particularly well for HTML files, since we don't support multiple files at a time, which is required for most HTML package apps.
